### DISCUSSION TITLE
Full text search option to the contains command

### DISCUSSION THREAD

**Discussion by tit** [cypress-io/cypress#16952] [Questions and Help] [answered]
Hi.

I have some HTML code:

```
<span>A_FOO</span>
<span>FOO</span>
```

And I want to search second span with FOO text content.

```
cy.contains('span', 'FOO') // I use strong string, not RegExp.
```

But Cypress returns first span with A_FOO, because A_FOO contains FOO substring.
I suggest adding the full text search option to the contains command options.

![image](https://user-images.githubusercontent.com/278604/122173605-92490600-ce8a-11eb-8faa-de68625d4acb.png)

**Comment by jennifer-shehane** [comment, marked as answer]
I would suggest using Regex to get the result you want. Adding this feature is not to be a high priority for us since it is already possible.

```js
cy.contains('span', /^FOO$/)
```

**Reply by tit** [reply]
@jennifer-shehane thank you!
